### PR TITLE
Gb/dropout bug

### PR DESCRIPTION
**Pull request by grantbuster**

### CODE REVIEW

**Comment by grantbuster** [comment]
interesting, you dont actually dropout data during training, only during prediction (validation and production predictions):

"indicating whether the layer should behave in training mode (adding dropout) or in inference mode (doing nothing)."

https://www.tensorflow.org/api_docs/python/tf/keras/layers/Dropout#call_arguments

**Review by dhetting — APPROVED** [review]
